### PR TITLE
[WIP] Added ReactiveCocoa and MVVM

### PR DESCRIPTION
**Pull request by pteasima**
ReactiveCocoa and MVVM

### CODE REVIEW

**Comment by pteasima** [comment]
It looks good to me in Visual Studio Code but is a little messed up here, will fix later.

File: Ingredients.md [hunk 1]
```diff
@@ -14,6 +14,100 @@ Set of Libraries we use during development
...
+us to structure the app in such a way, that all of the streams are flowing in the same direction.
+The Model layer is owned by the ViewModels, which publish the data in a view-friendly and observable way.
+The Views own the ViewModels and always update to their latest state.
+User input handling in the View layer is very light.
+The View usually just calls a Void method or an Action on the ViewModel and that's it.
+The ViewModel has full control over what happens next.
+No waiting for a result, no `updateView(model:)` calls,
+just knowing that if some ViewModel data changes as a result of this or any other action, it will be observed. 
+
+[//TODO: link na mvvm](http://google.com)
```
Comment (pteasima) [inline Ingredients.md]: @Dominoo you said you had an up to date MVVM example link?

Comment (Dominoo) [inline Ingredients.md]: simple MVVM example https://github.com/Yalantis/StarWars.iOS
does it look good already ? can I merge it ?

Comment (pteasima) [inline Ingredients.md]: fuck github closes the mr when I delete the branch. I will create a new one.

**Pull request closed without merging**